src/gazedetection/gaze_detection.py

```
@@ -0,0 +1,240 @@
+    try:
+        model = joblib.load('gaze_model.pkl')
+        print("--- 실시간 예측 모드 ---")
+    except FileNotFoundError:
+        print("오류: 학습된 모델 파일(gaze_model.pkl)을 찾을 수 없습니다.")
+        exit()
+
+cap = cv2.VideoCapture(0)
+
+while True:
+    ret, frame = cap.read()
+    if not ret: break
```

Ruff (0.13.3)

Multiple statements on one line (colon)

(E701)